src/Config/RuntimeConfigLoader.cs

```diff
@@ -64,6 +64,32 @@ public RuntimeConfigLoader(string? connectionString = null)
             {
                 config = config with { DataSource = config.DataSource with { ConnectionString = connectionString } };
             }
+
+            // For Cosmos DB NoSQL database type, DAB CLI v0.8.49 generates a REST property within the Runtime section of the config file. However
```

Comment:
this version details should not be a part of comment/ code summary

Reply:
I modified it to be more generic with +/- attributes.